query/security/ldap/src/test/groovy/org/codice/ddf/admin/ldap/discover/LdapTestBindSpec.groovy

```diff
@@ -166,12 +164,13 @@ class LdapTestBindSpec extends Specification {
         report.result().getValue()
     }
 
-
+    @Ignore
```

Comment:
Can we leave a TODO explaining why this is ignored? I assume it's because the ldap in memory ldap doesn't support md5?